test/nyc-bin.js

```diff
@@ -479,6 +479,23 @@ describe('the nyc cli', function () {
           done()
         })
       })
+
+      it('allows a sub-directory of files to be instrumented', function (done) {
```

Comment:
awesome \o/ thank you for the test. random aside, I think these integration tests are probably the approach I should have taken all along with nyc, I think I might start retiring some of the older crufty test files that are kicking around.